app/controllers/users_controller.rb

```diff
@@ -108,6 +109,7 @@ def filters
         render json: filters_json
       end
     end
+    render layout: 'without_sidebar'
```

Comment:
This render call has the same problem as the preferences one: there is already a render call in the JSON section so this would be a second render call, which is not allowed. The same solution applies - this line just needs to be inside the `format.html` section, after the `authenticate_user`.